tests/e2e/cucumber/features/smoke/spaces/participantManagement.ocis.feature

```diff
@@ -16,10 +26,12 @@ Feature: spaces participant management
       | name | id     |
       | team | team.1 |
     And "Alice" navigates to the project space "team.1"
-    And "Alice" adds following users to the project space
-      | user  | role   |
-      | Brian | editor |
-      | Carol | viewer |
+    And "Alice" adds following members to the project space
+      | user     | role   | kind  |
```

Comment:
What do you think about naming it "name" instead of user? I find "user" is a bit confusing when we're dealing with groups as well here.

```suggestion
      | name     | role   | kind  |
```